server/monitor.go

```diff
@@ -2231,6 +2232,11 @@ func (s *Server) Leafz(opts *LeafzOptions) (*Leafz, error) {
 			leafnodes = append(leafnodes, lni)
 		}
 	}
+
+	if len(leafnodes) == 0 {
```

Comment:
Probably should be `if leafnodes == nil`

Reply:
nods - I moved the make out of the if though so now there's no need to check it, its always right this is better.

Reply:
Thanks..